src/libm-tester/CMakeLists.txt

```diff
@@ -34,8 +42,11 @@ macro(test_extension SIMD)
     add_dependencies(${TARGET_IUT${SIMD}} ${TARGET_HEADERS})
     add_dependencies(${TARGET_IUT${SIMD}} ${TARGET_LIBSLEEF})
 
-    add_test(NAME ${TARGET_IUT${SIMD}}
-      COMMAND ${TARGET_TESTER} ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${TARGET_IUT${SIMD}})
+    list(FIND SKIP_TEST ${SIMD} INDEX_SKIP_TEST)
+    if (INDEX_SKIP_TEST EQUAL -1)
```

Comment:
What is going on here? Why do you want to turn off the regular tester when you are building the tester3 executables? Is this something you just did for debug, to skip running the iutsimd tests because it takes too long for them to complete?

Reply:
Since tester3 checks if the computation of the given two implementations are all same, it should be enough to check only one of them using the original tester. I am going to add more implementations with different bit lengths, and I think testing will take too much time without doing like this.

Reply:
As you see, there is an option SLEEF_TEST_ALL_IUT. By turning this on, all implementations are checked by the original tester. I believe it is reasonable to test all of them only occasionally.